docs/proposal/2_1_ocg_objectives.tex

```diff
@@ -1,26 +1,21 @@
-The main objectives have been separated into three categories.
+The main objectives are sorted by the client's priority in descending order.
 
-\textbf{Machine Learning Implementation on the FPGA}
-\begin{itemize}
-  \item The FPGA and its connected hardware receives usable video data.
-  \item Adequately implemented datapath and or processor to facilitate data-flow.
-  \item Adequately implemented machine learning model such as CNN, RNN, or YOLO in hardware.
-  \item Video can be processed in real time with help of reduced frame rates and or resolution.
-\end{itemize}
+\textbf{Integration of Electronics with the Drone}:
+Since the ultimate deliverable consists of a highly mobile computing platform,
+our uttermost most important objective is a successful integration of computation and processing hardware with the drone.
+The equipment for capturing vidoe data, perform processing, and trasmitting digital data should be reasonbly compact and easily deployable.
```

Comment:
```suggestion
The equipment for capturing video data, performing processing, and transmitting digital data should be reasonably compact and easily deployable.
```

Reply:
Lmao 3x in one sentence. Guess I shorted